src/server/gc_worker.rs

```diff
@@ -1287,8 +1288,21 @@ mod tests {
     use futures::Future;
     use kvproto::metapb;
     use std::collections::BTreeMap;
+    use std::mem;
```

Comment:
This import is not needed because it is imported from the parent by `use super::*;`.